content/md/administration/catalog-volume-monitoring_fr.md

```diff
@@ -0,0 +1,72 @@
+---
+id: catalog-volume-monitoring
+themes: administration, akeneo-concepts, catalog-settings 
+title: Controllr la **taille de votre catalogue** (since 2.3)
+popular: false
+related: what-is-a-product-value, what-is-an-attribute, what-is-a-family, what-is-a-channel, what-is-a-locale, what-is-a-category
+---
+
+# Un écran dédié
+
+Un écran est disponible dans le PIM afin de controller la taille du catalogue sur plusieurs axes.
+
+Cet écran contient par exemple le nombre de familles ou le nombre moyen d'attributs par famille.
+
+![Catalog volume monitoring dashboard](../img/Activity_CatalogVolumeMonitoring_fr.png)
+
+::: warning
+Cet écran n'est disponible que depuis la version 2.3 du PIM.
+:::
+
+# Où est-ce?
+
+Voici les étapes afin d'accéder à ce superbe (;)) écran:
+
+1. Cliquer sur le menu `Activité`
+2. Cliquer sur le sous-menu `Volume du catalogue`
+3. Et voilà !
+
+# Comment comprendre ce dashboard ?
+
+## Les différentes sections
+
+L'écran `Volume du catalogue` est composé de plusieurs sections.
+
+### Le nombre de valeurs de produit
+
+La première valeur affichée dans le widget représente le nombre total de [valeurs de produit](/articles/what-is-a-product-value.html) contenues dans le PIM.
+C'est un indicateur de la valeur des données présentes dans votre catalogue produit. 
+
+![Widget du nombre de valeurs de produit](../img/Activity_CatalogVolumeMonitoring_PvWidget_fr.png)
+
+Nous vous encourageons à surveiller cette valeur régulièremeent. Plus le nombre de valeurs de produit est important, plus votre PIM vous est utile !
+
+Nous avons écris un [article](https://medium.com/akeneo-labs/because-your-product-catalog-typology-matters-e1a9af4c33e0) en anglais afin d'expliquer pourquoi le nombre de valeurs produit est une métrique intéressante pour évaluer la taille de votre catalogue. Ne soyez pas effrayé par la taille de cet article ! Il mettra en lumière toute vos interrogations. Vous ne parlerez plus de la taille de votre catalogue de la même façon ;)
```

Comment:
valeurs de produit

Reply:
`Il mettra en lumière toute vos interrogations` is a little bit too much.
What about `, il répondra à toute vos questions`

Reply:
done